{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/data/builder.py

```diff
@@ -0,0 +1,129 @@
+"""Modularized functions for building project data artifacts.
```

Comment:
Add some detail to the module docstring here.  This module is an abstraction around the load process in our artifact building ETL pipeline.  It should strive to be declarative so it's easy to see what is put into the artifact and how. Etc.

Reply:
Done.